src/routes/TraderForm/index.tsx

```diff
@@ -20,9 +27,19 @@ import {
 } from './TraderForm.styled';
 
 export const TraderFormRoute: React.FunctionComponent = () => {
+  const dispatch = useAppDispatch();
   const { form } = useParams();
   const { aMM, loading, error, idle } = useSwapFormAMM();
 
+  useEffect(() => {
+    if (!aMM) {
+      return;
+    }
+    void dispatch(getFixedRateThunk());
```

Comment:
I think best place for these dispatch(s) is inside the useSwapFormAMM. The logic being that the hook useSwapFormAMM should initialize the AMM for us. Later on when wallets are connected/disconnected we should also do dispatching from the useSwapFormAMM. Can you refactor move them over there?

Reply:
[957310c](https://github.com/Voltz-Protocol/voltz-ui/pull/571/commits/957310cc8a0b03d2680b71d8f8ec5e98803f7150)